File: app/src/main/java/me/echeung/moemoekyun/client/socket/response/ConnectResponse.java
package me.echeung.moemoekyun.client.socket.response;

import lombok.Getter;
import me.echeung.moemoekyun.client.model.User;

@Getter
public class ConnectResponse extends BaseResponse {
    private Details d;

    @Getter
    public static class Details {
        private int heartbeat;
        private String message;
        private User user;
    }
}

File: app/src/main/java/me/echeung/moemoekyun/client/api/socket/response/ConnectResponse.kt
package me.echeung.moemoekyun.client.api.socket.response

import me.echeung.moemoekyun.client.model.User

class ConnectResponse : BaseResponse() {
    val d: Details? = null

    class Details {
        val heartbeat: Int = 0
        val message: String? = null
        val user: User? = null
    }
}
